etl/src/main/java/org/pih/hivmigration/etl/sql/ExamLabResultsMigrator.groovy

```diff
@@ -49,10 +49,7 @@ class ExamLabResultsMigrator extends ObsMigrator {
             SELECT
                 he.source_encounter_id,
                 concept_uuid_from_mapping('CIEL', '5497'),
-                CASE
-                    WHEN is_number(result) THEN TRIM(result)
-                    WHEN result REGEXP '^\\d*\\s*mm$' THEN extract_number(result)
-                    END
+                extract_number(result)
```

Comment:
Is this based on the comments from @mseaton  earlier today, or is it unrelated? If it's related, shouldn't we be changing the WHERE clause here too to skip the ones that have "mm"?